floodwatch/src/js/components/Generate.js

```diff
@@ -67,3 +108,68 @@ export class Generate extends Component {
     )
   }
 }
+
+function decodeFilterRequestItem(filter: FilterRequestItem): Array<Filter> {
+  const keys = _.keys(filter)
+  const isPersonal = (_.indexOf(keys, 'personal') > -1);
+
+  if (isPersonal) {
+    return [{
+      name: 'data',
+      logic: 'or',
+      choices: ['You']
+    }]
+  }
+
+  let optionsArr = []
```

Comment:
Another meta comment that doesn't need to be addressed now but for the future: you should prefer to use `const` over `let` wherever possible. By using `let` variables only for times you need to change the assignment of the variable, it's clearer which variables are intended to be changed vs not.